### DISCUSSION TITLE
No more omitting useful code

### DISCUSSION THREAD

**Discussion by pixelmixel** [cline/cline#478] [Feature Requests]
Very often I am prompted with a code solution that says "leave the code as it is" or something of that nature yet VS Code will overwrite this comment as the code. Leaving me to undo the code, and re-ask Claude/Cline to "write it out fully, no omissions" which is very annoying and expensive. I'd love that if the code comes back <leave the code as it is> the VS Code actually leaves it as it is or it's automatically disabled from being overwritten when I hit save.

**Comment by detroittommy879** [comment]
I keep having this problem on Gemini 1.5 Pro, Claude Sonnet, lots of LLMs keep doing it, maybe it is a prompting issue and no one has figured out how to make them stop doing it? I tried "yelling" with all caps, and lots of !'s... which seemed to help, but i haven't consistently tried it enough to know if it is reliable. I will try to keep doing that and see if it does it again.

**Reply by TomLucidor** [reply]
The central problem is that most Cursor hackers recommend having a user requirements or documentation MD file to help steer the LLM into not erasing useful code, so there has to be a way to put up with this (see also TDD https://github.com/cline/cline/discussions/535 and Architect mode https://github.com/cline/cline/discussions/420)

**Comment by bilguunchinzorigEPFL** [comment]
For me this is causing the major annoyance because most of the time it removes whole useful code on minor changes. Plus instead of generating whole file making small changes would cost much less. Is there any update on this feature?
